mina-archive/scripts/db-bootstrap-command.sh

```diff
@@ -118,7 +118,7 @@ fetch_extra_sql_files() {
 
   for URL in $URLS; do
     info "Downloading $URL"
-    curl "$URL" -sO
+    curl "$URL" -sO --fail || { error "Unable to download the URL"; return 1; }
```

Comment:
**suggestion** Could we print out the URL, so one could read it from stderr?

```
curl "$URL" -sO --fail || { error "Unable to download content from URL $URL"; return 1; }
```

Reply:
Also why do we have to do `return 1` at the end? 

The `error` function already returns exit code 1.

Reply:
Yes, the print of the URL is done the line above.

For the return, it is related to this: https://askubuntu.com/questions/1450617/set-e-subshell-doesnt-exit-on-the-first-command-that-fails-if-i-capture-the-er.